### ISSUE TITLE
Font Styles Not Loading  -- w:ascii is not present only W:cs

### ISSUE THREAD

**Issue by Paperclip-Systems** [PHPOffice/PHPWord#1235]
This is:

- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpword)

# Expected Behavior

The fonts to load even if they are not listed as w:ascii, and retain default styling.  

### Current Behavior

Fonts not listed as w:ascii are ignored.  The file was created in word 2016 and saved as a docx.  The style xml does not have the w:ascii tag, so the fonts are not loading causing it to loose its times new roman font setting.  Additionally the paragraph styling is not retaining for the default style (in this case it looses it center formatting and goes to left set)

### Failure Information

See above.  The problem occurs when loading the docx file and writing it out again.

### How to Reproduce

Please provide a code sample that reproduces the issue.
```php
require_once './vendor/autoload.php';

use PhpOffice\PhpWord\PhpWord;
use PhpOffice\PhpWord\Settings;

Settings::loadConfig();
$phpWord = \PhpOffice\PhpWord\IOFactory::load("living-will.docx", "Word2007");
$temp_name="/tmp/temp.docx";
$phpWord->save($temp_name, "Word2007", false);
header('Content-Transfer-Encoding: Binary');
header('HTTP/1.1 200 OK');
header('Status: 200 OK');
header('Content-Type: application/octet-stream');
header('Content-Type: application/force-download');
header("Content-Description: File Transfer");
header('Content-Disposition: attachment; filename="living-will.docx');
header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
header("Cache-control: private");
header('Pragma: private');
readfile($temp_name);
unlink($temp_name);

```

### Context
[living-will.docx](https://github.com/PHPOffice/PHPWord/files/1599374/living-will.docx)
[living-will-Output.docx](https://github.com/PHPOffice/PHPWord/files/1599377/living-will-Output.docx)

* PHP version: 7
* PHPWord version: latest dev branch

**Comment by troosan** [comment]
@Paperclip-Systems please have a look at the PR I just submitted.
Anyway, from the looks of the document, it looks to me that what you need is the TemplateProcessor, not the IOFactory::load()

**Comment by Paperclip-Systems** [comment]
Troosan, I looked over the submitted PR and it looks like it should resolve the issue.  Once it is merged in I will try it out.  The document I submitted would work with the template processor, and I generally agree with that application.  In this case I selected the document because I knew it could be used to replicate the problem and it was convenient because the xml files were not all that complicated.  

Thank you for looking into this issue.  I am not familiar enough with the data structure and class method interactions to fix it myself.
